eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -81,4 +92,42 @@ public Input get(final String name) throws MalformedURLException {
         return new InputOf(url);
     }
 
+    /**
+     * Hash of head master.
+     * @return SHA of commit
+     * @throws IOException if fails
+     * @checkstyle NonStaticMethodCheck (20 lines)
+     */
+    private String getSha() throws IOException {
+        String sha = "master";
+        final String command = "curl -o tags.txt https://raw.githubusercontent.com/"
+            .concat("objectionary/home/gh-pages/tags.txt");
+        try {
+            Runtime.getRuntime().exec(command).waitFor();
+        } catch (final InterruptedException exception) {
+            Logger.info(this, exception.toString());
+        }
+        final File file = new File(System.getProperty("user.dir").concat("/tags.txt"));
+        try {
```

Comment:
@yasamprom I think it would be better to use try-with-resources construction here.